package.json

```diff
@@ -17,7 +17,10 @@
   "scripts": {
     "start": "grunt",
     "test": "grunt test",
-    "postinstall": "bower install --config.interactive=false"
+    "postinstall": [
+      "bower install --config.interactive=false",
+      "./node_modules/protractor/bin/webdriver-manager update"
```

Comment:
If we use that package, would we still be required to run the server with our protractor test task? I couldn't tell from the documentation.

Either way, `grunt-protractor-webdriver` looks promising. I'm gonna take a shot at testing it.